doc/user/release-notes.md

```diff
@@ -28,6 +28,8 @@ For information about available versions, see our [Versions page](../versions).
 
 - Support naming columns when creating CSV sources.
 
+- Support jsonb_agg().
```

Comment:
nit, tioli: can you link this to the functions and operators documentation (however close to the actual heading you can get), and rephrase as "Support jsonb_agg() aggregate function."?

Reply:
nit: can you surround jsonb_agg() in backticks for code formatting?